contrib/format-maprdb/src/main/java/org/apache/drill/exec/store/mapr/db/json/MaprDBJsonRecordReader.java

```diff
@@ -517,6 +388,15 @@ private static FieldPath getFieldPathForProjection(SchemaPath column) {
     return new FieldPath(child);
   }
 
+  public static boolean includesIdField(Collection<FieldPath> projected) {
+    return Iterables.tryFind(projected, new Predicate<FieldPath>() {
```

Comment:
@vdiravka could you elaborate on this a bit ?  I haven't done much with lambda expressions in Java 8, but if you show how to rewrite this statement and assuming it has an advantage over the existing implementation, I would be happy to.  Thanks.

Reply:
Since this is a syntactic modification, I am inclined to leave it as-is .. partly because I might cause an unintentional side effect/regression and partly because such functional utilities appear elsewhere in the Drill and Calcite code.  Ideally, if we want to move to Lambdas, we should re-visit all such implementations.

Reply:
Agreed. 
Vova was doing that, but has not finished in the result.